_rest_routes.inc.php

```diff
@@ -886,6 +914,31 @@
         $return = (new FhirPersonRestController())->getOne($uuid);
         RestConfig::apiLog($return);
         return $return;
+    },
+    // Bulk FHIR api endpoints
+    'GET /fhir/Group/:id/$export' => function ($groupId, HttpRestRequest $request) {
+        RestConfig::authorization_check("admin", "users");
+        $fhirExportService = new FhirExportRestController();
+        $fhirExportService->sendExportHeaders();
+        $return = $fhirExportService->processExport($_GET, 'Group');
+        RestConfig::apiLog($return);
+        return $return;
+    },
+    'GET /fhir/Patient/$export' => function (HttpRestRequest $request) {
```

Comment:
looks like this same endpoint is repeated above (guessing you meant to just have it here in this section)

Reply:
what is a quick example of what $export variable represents (I tried to search for $export in this PR but my brain couldn't figure it out :) )

Reply:
So the FHIR specification uses the $ sign to mark anything that is in FHIR terminology an [operation](https://www.hl7.org/fhir/operations.html).  Rather than mapping to a Create/Read/Update/Delete request it's supposed to be something that is more of a Remote Procedure Call (RPC) to tell the server to execute something.  In this case the $export is representing the fact that this is the **export-patient** FHIR RPC call to fire off an asynchronous request to export all of the patient data.